x-pack/legacy/plugins/maps/common/constants.js

```diff
@@ -30,7 +30,7 @@ export const GEOJSON_FILE = 'GEOJSON_FILE';
 
 export const DECIMAL_DEGREES_PRECISION = 5; // meters precision
 export const ZOOM_PRECISION = 2;
-export const DEFAULT_ES_DOC_LIMIT = 2048;
+export const DEFAULT_ES_DOC_LIMIT = 10000;
```

Comment:
can we rename this constant to something like 'ES_SIZE_LIMIT` and reuse for the aggs as well:

- https://github.com/thomasneirynck/kibana/blob/7ac8e4d9ccc2a9e064d8a5621cd62453d1006164/x-pack/legacy/plugins/maps/public/shared/layers/sources/es_join_source.js#L174
- https://github.com/thomasneirynck/kibana/blob/a323753b5ce2fec267c3b59dcd073c37c595deb6/x-pack/legacy/plugins/maps/public/shared/layers/sources/es_search_source/es_search_source.js#L165

It makes sense for them to be identical. (conceptually, cannot join more tha